sale_line_name_option/models/res_company.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    no_product_code_in_sale_line_name = fields.Boolean(default=True)
```

Comment:
To avoid causing hiccups in tests.
```suggestion
    no_product_code_in_sale_line_name = fields.Boolean(default=False)
```